### ISSUE TITLE
Installation and Documentation Improvements Needed

### ISSUE THREAD

**Issue by huangnengCSU** [LappalainenLab/lorals#15]
Dear Developer,

Could someone provide clear installation instructions, including the required Python version and dependencies? Alternatively, releasing a Docker image could greatly simplify installation.

A well-organized documentation would be extremely helpful for users. 

Thank you!

**Comment by margaux-david** [comment]
Hi @huangnengCSU, have you figured out a way to install the package and get it to work? I am struggling with the same error and cannot seem to make it work. When running like `calc_ase --help`, I get this output: 
`NameError: name '__LORALS_SETUP__' is not defined` [...]
and `AttributeError: scipy.stats.stats has no attribute KstestResult; furthermore, scipy.stats.stats is deprecated and will be removed in SciPy 2.0.0.` Did you struggle with a similar error? How did you get it to work ? Thank you so much!

**Comment by huangnengCSU** [comment]
No, I haven't resolved this issue. It seems like no one is maintaining this tool.

**Comment by fbyukgo** [comment]
Hey @margaux-david, @huangnengCSU,

I'm not sure if this is still a relevant issue, but the workaround I found is:

Install scipy separately in your conda environment before running the LORALS installation script, as the LORALS installation somehow fails to install it properly:
`conda install scipy=1.7.3`

Then run the installation script:
`python setup.py install`

After that, check the installation by running:
`calc_ase --help`

Hopefully this works for you as well!
